cmd/vic-machine/inspect/inspect.go

```diff
@@ -25,6 +25,8 @@ import (
 	"github.com/Sirupsen/logrus"
 	"gopkg.in/urfave/cli.v1"
 
+	"github.com/vmware/govmomi/find"
```

Comment:
Nit: we've included govmomi imports in the same group as vic imports in most of our code - we could do the same here.